src/debt_order/index.ts

```diff
@@ -44,22 +58,42 @@ export class DebtOrder {
      */
     public static async create(dharma: Dharma, params: DebtOrderParams): Promise<DebtOrder> {
```

Comment:
Need to also update the docs for this method

Reply:
Done